README.md

```diff
@@ -0,0 +1,65 @@
+
+# OJP - *O*pen API for distributed *J*ourney *P*lanning
+
+## Content
+This repository contains the XML Schema, examples and documentation tables for the "**O**pen API for distributed **J**ourney **P**lanning" published as *"CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"*.
+
+OJP consists of a set of services that can be addressed using HTTP requests.
+
+The main services to request information are:
+* Place information
+* Stop events (arrival and departure boards)
+* Searching for trips
+* Information about a given trip
+* Information about fares for trips
+* Changing of trips
+* Refining of trips
+* Availability for trip legs
+* Information about lines
+
+OJP systems interacting with each other additionally use:
+* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
+* Using a multi point trip service (n origins and m destinations)
+
+## Standard document
+OJP is a CEN standard. Obtain a copy of the standard document ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7) at [CEN](https://www.cen.eu).
+
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
+
+## Releases
+* [OJP 1.0 branch](https://github.com/VDVde/OJP/tree/release/1.0)
+  * [OJP 1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
+  * [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bug fix release
+* OJP 2.0 in preparation. See the following [branch](https://github.com/VDVde/OJP/tree/develop).
+
+## Github Repository
+
+Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
+
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The master branch is currently set to OJP 1.0.1
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
+* We work with release-branches
+* PR should always be created with feature/<id> as a branch.
```

Comment:
You did not create a feature branch for this change itself..

Reply:
I know. Last one :-)